### ISSUE TITLE
Add a new sample for kraken output

### ISSUE THREAD

**Issue by zuphilip** [kba/ocr-fileformat-samples#10]
It seems that the hocr output changed lately in kraken: remove cuts by x_bboxes see https://github.com/mittagessen/kraken/commit/da29e34aaa7538bbb81146866bd9c6baeb7d49ad . It would be good to have a sample output of the new kraken version. CC @mittagessen

**Comment by mittagessen** [comment]
Yeah, the old cuts based grapheme boxes were the result of me misreading the hOCR specification. The current output passes the validator and should be a lot easier to parse.

An example page containing some Syriac text is available at:
http://br.unchti.me/13.png http://br.unchti.me/13.hocr

I've recently added vertical script support. If desired I could probably find some examples for that, too.

**Comment by zuphilip** [comment]
Thank you for the nice example! One can make now a PR out of this.

I just saw that you are also using `ocrx_word` for spaces:

![kraken-hocr](https://cloud.githubusercontent.com/assets/5199995/25681419/89090834-3054-11e7-928c-9b22d4df010b.png)

> I've recently added vertical script support. If desired I could probably find some examples for that, too.

Yes, I would be interesting to have a sample for this as well. I guess @kba will agree on this?

**Comment by mittagessen** [comment]
I guess it would be possible to put them into separate ``ocrx_cinfo`` elements but I prefer uniformity and there is this bit about ``ocrx_cinfo`` nesting into an ``ocrx_line`` which we don't use. But then words aren't really defined so I just treat them as alternating string and whitespace elements (similar to ALTO).

Vertical and BiDi samples are probably interesting for seeing how stuff breaks. The software in your screenshot misaligns RTL text to the left; that's a side-effect of the nested spans which trip the BiDi detection in most browsers. Unfortunately there is, AFAIK, no truly typographically neutral/semantic only markup element in HTML to circumvent this without setting general text direction manually.

**Comment by kba** [comment]
Sure, up-to-date kraken output would be great.

> Vertical and BiDi samples are probably interesting for seeing how stuff breaks. The software in your screenshot misaligns RTL text to the left

Indeed, text direction is not something I considered yet for hocrjs.
